Automattic/newspack-ads
# feat: update block icon

This PR:
* Uses a custom icon instead of Gutenberg's
* Removes the need for the `@wordpress/icons` package

How to test:

1. Open Gutenberg
2. Check the current icon of the Ad Unit block
3. Switch to this branch
4. Check again
